packages/config/config/devices/0x024f/fwc01.json

```diff
@@ -0,0 +1,227 @@
+{
+	"manufacturer": "Smartly AS",
+	"manufacturerId": "0x024f",
+	"label": "FWC01",
+	"description": "Wheel Controller",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x1003"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "4",
+			"label": "Safety Activate Delay",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 0
+		},
+		{
+			"#": "5",
+			"label": "Safety OFF Period",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 600
+		},
+		{
+			"#": "6",
+			"label": "Safety ON Period",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 300
+		},
+		{
+			"#": "10",
+			"label": "Dimming speed",
+			"description": "Dimmer Speed in 1/100th of a Second",
+			"unit": "/100s",
+			"valueSize": 2,
+			"minValue": 30,
+			"maxValue": 32767,
+			"defaultValue": 300
+		},
+		{
+			"#": "11",
+			"label": "Dimmer mode",
+			"description": "Dimmer Operation Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
```

Comment:
What do these values mean? There should probably be options describing them.

Reply:
Yes, I'll correct that. Once more I used another config as a template, which had defined this config like this (see fms01.json in the same folder as this).